sechub-developertools/scripts/sechub-api.sh

```diff
@@ -35,6 +37,7 @@ profile_details <profile-id> - Show details of execution profile <profile-id> (e
 profile_list - List all existing execution profiles (json format)
 project_assign_profile <project-id> <profile-id> - Assign execution profile <profile-id> to project <project-id>
 project_assign_user <project-id> <user-id> - Assign user to project (allow scanning)
+project_create <project-id> <owner> ["<project short description>"] - Create a new project; short description is optional
```

Comment:
```suggestion
project_create <project-id> <owner> ["<project short description>"] - Create a new project with project name <project-id> and owner <owner>. The short description is optional.
```

Reply:
I tried to keep it small so no line break occurs because of long lines.
Your proposal would imho make the line too long although complete sentences are nice.
The current line is already 122 columns wide.

Reply:
How about this? Using a semi-colon interrupts the reading.

```suggestion
project_create <project-id> <owner> ["<project short description>"] - Create a new project. The short description is optional.
```